src/scripts/main.js

```diff
@@ -1 +1,67 @@
 'use strict';
+
+const promise1 = new Promise((resolve, reject) => {
+  document.addEventListener('click', () => {
+    resolve('First promise was resolved');
+  });
+
+  setTimeout(() => reject(new Error('First promise was rejected')), 3000);
+});
+
+function newMessage(text, className, classModificator) {
+  const divMessage = document.createElement('div');
+
+  divMessage.className = className;
+  divMessage.classList.add(classModificator);
+  divMessage.textContent = text;
+  document.body.append(divMessage);
+}
+
+promise1
+  .then((text) => {
+    newMessage(`${text}`, 'success', 'success--1');
+  })
+  .catch((text) => {
+    newMessage(`${text}`, 'warning');
+  });
+
+const promise2 = new Promise((resolve) => {
+  document.addEventListener('click', () => {
+    resolve('Second promise was resolved');
+  });
+
+  document.addEventListener('contextmenu', (ev) => {
+    ev.preventDefault();
+    resolve('Second promise was resolved');
+  });
```

Comment:
use `mousedown` event
and then e.button === 0 or 2 to check was it right button clicked